batavia/core/builtins.js

```diff
@@ -1415,5 +1425,5 @@ for (var fn in batavia.builtins) {
     batavia.builtins[fn].__python__ = true;
 }
 
-batavia.builtins.None = null;
+batavia.builtins.None = batavia.types.NoneType;
```

Comment:
This seems odd to me. In CPython, `None` is an instance of `NoneType`, but not equal to it.

Also, I think when making this, we'll have a lot of extra cleanup to do. (For example, searching for `== null` in the codebase has dozens of hits that appear to be `None` checks.